.github/workflows/push_tests.yml

```diff
@@ -3,6 +3,9 @@ name: Built-in Tests for Push (Xpress in Ubuntu 18.04)
 on:
 
   push:
+    paths-ignore:
```

Comment:
@hdunham could this potentially cause issues if migrations are not handled properly on a push that happens to only include README.md and/or CHANGELOG.md?